.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+- List changes with brief descriptions
+  - Closes issue #
+
+## Submitter Checklist
+
+- [ ] Verify all tests pass
+  - Run tests with `docker-compose -f test-compose.yml -p onitest up test`
+- [ ] Update `README.md` and `docs/` as necessary
+  - If a `manage.py` command is added, deleted, or modified its help text must
+    be valid and it should be documented in detail in
+    `docs/advanced/admin-commands.md`
+- [ ] Update `CHANGELOG.md`
+  - Describe change(s) in appropriate section(s)
+  - List self in Contributors section
+  - If a release, replace `[Unreleased]` with version and update compare link
+- [ ] Resolve merge conflicts
+- [ ] @mention individual(s) you would like to review the PR
```

Comment:
Why not just add them to the reviewers list?  Or does a @mention do this automatically?

Reply:
I saw this as a suggestion in another PR template when I was researching. I didn't know how to test it (without making a whole separate GitHub account etc), but my guess was that for non-maintainers they might not be able to add reviewers to the PR. I'm guessing the @mention alone won't add an individual to the reviewers list, but it will draw the attention of that user through notifications. If non-maintainers can't request a review through the UI otherwise, I figured this is a good backup.

If it turns out that's wrong, we can axe/update that part of the template later.